deploy/ansible/roles/local/ansible-users/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
also this. gitignore?

Reply:
this file is a part of an external module. Some functionality could be added in future